### PR TITLE
Forms 9273- handling repeatability caching  and setFocus issue when minOccur is greater than 1 for any panel layout

### PR DESCRIPTION
**Pull request by deepprakash345**
Forms 9273- handling repeatability caching when minOccur is greater than 1 for any panel, handling setfocus when minoccur > 1

Validations done:- test cases added and manual testing also done

### CODE REVIEW

File: ui.tests/test-module/specs/accordion/accordion.minoccur.spec.js [hunk 1]
```diff
@@ -38,5 +60,22 @@ describe("Min Occur Repeatability Tests in Accordion", () => {
...
+            getAccordionPanelsAtIndex(3).should('not.have.class', 'cmp-accordion__panel--expanded');
+            getAccordionButtonsAtIndex(3).should('not.have.class', 'cmp-accordion__button--expanded');
+            getAccordionPanelsAtIndex(4).should('not.have.class', 'cmp-accordion__panel--expanded');
+            getAccordionButtonsAtIndex(4).should('not.have.class', 'cmp-accordion__button--expanded');
+            getAccordionPanelsAtIndex(5).should('not.have.class', 'cmp-accordion__panel--expanded');
+            getAccordionButtonsAtIndex(5).should('not.have.class', 'cmp-accordion__button--expanded');
+        })
+    })
+
     //TODO Add setfocus test minoccur case
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/accordion/accordion.minoccur.spec.js:80]: remove comment

File: ui.tests/test-module/specs/tabsontop/tabsontop.minoccur.spec.js [hunk 2]
```diff
@@ -38,5 +38,22 @@ describe("Min Occur Repeatability Tests in Tabs on Top", () => {
...
+            cy.get(".cmp-tabs__tab").eq(3).should('not.have.class', 'cmp-tabs__tab--active');
+            cy.get(".cmp-tabs__tabpanel").eq(3).should('not.have.class', 'cmp-tabs__tabpanel--active');
+            cy.get(".cmp-tabs__tab").eq(4).should('not.have.class', 'cmp-tabs__tab--active');
+            cy.get(".cmp-tabs__tabpanel").eq(4).should('not.have.class', 'cmp-tabs__tabpanel--active');
+            cy.get(".cmp-tabs__tab").eq(5).should('not.have.class', 'cmp-tabs__tab--active');
+            cy.get(".cmp-tabs__tabpanel").eq(5).should('not.have.class', 'cmp-tabs__tabpanel--active');
+        })
+    })
+
     //TODO Add setfocus test minoccur case
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/tabsontop/tabsontop.minoccur.spec.js:58]: remove comment

File: ui.tests/test-module/specs/wizard/wizard.minoccur.spec.js [hunk 3]
```diff
@@ -38,5 +38,96 @@ describe("Min Occur Repeatability Tests in Wizard", () => {
...
+            cy.get(".cmp-adaptiveform-wizard__tab").eq(3).should('not.have.class', 'cmp-adaptiveform-wizard__tab--active');
+            cy.get(".cmp-adaptiveform-wizard__wizardpanel").eq(3).should('not.have.class', 'cmp-adaptiveform-wizard__wizardpanel--active');
+            cy.get(".cmp-adaptiveform-wizard__tab").eq(4).should('not.have.class', 'cmp-adaptiveform-wizard__tab--active');
+            cy.get(".cmp-adaptiveform-wizard__wizardpanel").eq(4).should('not.have.class', 'cmp-adaptiveform-wizard__wizardpanel--active');
+            cy.get(".cmp-adaptiveform-wizard__tab").eq(5).should('not.have.class', 'cmp-adaptiveform-wizard__tab--active');
+            cy.get(".cmp-adaptiveform-wizard__wizardpanel").eq(5).should('not.have.class', 'cmp-adaptiveform-wizard__wizardpanel--active');
+        })
+    })
+
     //TODO Add setfocus test minoccur case
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/wizard/wizard.minoccur.spec.js:58]: remove comment

**Review by sakshi-arora1 — APPROVED** [review]

File: ui.tests/test-module/specs/accordion/accordion.minoccur.spec.js (hunk 1 above)
Comment (deepprakash345) [reply ui.tests/test-module/specs/accordion/accordion.minoccur.spec.js]: done

File: ui.tests/test-module/specs/tabsontop/tabsontop.minoccur.spec.js (hunk 2 above)
Comment (deepprakash345) [reply ui.tests/test-module/specs/tabsontop/tabsontop.minoccur.spec.js]: done

**Pull request merged**